ingest/expression_files/expression_files.py

```diff
@@ -103,7 +103,7 @@ def is_raw_count_file(study_id, study_file_id, client):
         # If study files does not have document expression_file_info
         # field, "is_raw_count_files", will not exist.:
         if "expression_file_info" in study_file_doc.keys():
-            return study_file_doc["expression_file_info"]["is_raw_counts"]
+            return study_file_doc["expression_file_info"]["is_raw_count_files"]
```

Comment:
This is incorrect - the existing attribute of `is_raw_counts` is what's in the model.